components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/dto/PolicyStoreDTO.java

```diff
@@ -37,6 +37,10 @@ public class PolicyStoreDTO {
 
     private boolean setActive;
 
+    private String version;
+
+    private String policyType;
```

Comment:
Do we need this anymore. There are no usages for the policy type. @Niru-D

Reply:
Removed.